skyportal/handlers/api/internal/plot.py

```diff
@@ -6,23 +6,37 @@
 # TODO this should distinguish between "no data to plot" and "plot failed"
 class PlotPhotometryHandler(BaseHandler):
     @auth_or_token
-    def get(self, source_id):
-        docs_json, render_items, custom_model_js = plot.photometry_plot(source_id)
+    def get(self, source_or_candidate_id):
```

Comment:
The changes to this file account for the fact that plot data can be associated with either a source and a candidate, and some syntax changes (`black`)